account_invoice_transmit_method/models/account_move.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017-2020 Akretion France (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    transmit_method_id = fields.Many2one(
+        "transmit.method",
+        string="Transmission Method",
+        tracking=True,
+        ondelete="restrict",
+        domain="['|', '&', ('customer_ok', '=', transmit_method_domain_sale),"
+        "('customer_ok', '=', True),"
+        "'&', ('supplier_ok', '=', transmit_method_domain_purchase),"
+        " ('supplier_ok', '=', True)]",
+    )
+    # Field used to match specific invoice transmit method
+    # to show/display fields/buttons, add constraints, etc...
+    transmit_method_code = fields.Char(
+        related="transmit_method_id.code", store=True, string="Transmission Method Code"
+    )
+    transmit_method_domain_sale = fields.Boolean(
+        compute="_compute_transmit_method_domain", default=True
+    )
+    transmit_method_domain_purchase = fields.Boolean(
+        compute="_compute_transmit_method_domain", default=True
```

Comment:
Useless default value.
```suggestion
        compute="_compute_transmit_method_domain"
```